.github/workflows/cpp-linter.yml

```
@@ -3,15 +3,15 @@ name: cpp-linter
 on:
   push:
   pull_request:
-    types: [opened]
+    # types: [opened]
 
 jobs:
   cpp-linter:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: C/C++ Lint Action
-        uses: shenxianpeng/cpp-linter-action@master
+      # - uses: shenxianpeng/cpp-linter-action@master
+      - uses: 2bndy5/cpp-linter-action@parse-better
```

this should be changed back when ready
```suggestion
      - uses: shenxianpeng/cpp-linter-action@master
```